usr/share/rear/output/PXE/default/820_copy_to_net.sh

```diff
@@ -15,6 +15,19 @@ case "$scheme" in
     (fish|ftp|ftps|hftp|http|https|sftp)
         LogPrint "Transferring PXE files to $OUTPUT_URL"
         for result_file in "${RESULT_FILES[@]}" ; do
+            # OUTPUT_URL for lftp transfers will look something like:
+            # "<fish|ftp|ftps|hftp|http|https|sftp>://<host_name>/<destination_directory>"
+            # Using `cut' for printing 4th field with "/" delimiter (.. | cut -d/ -f4-),
+            # will strip leading "<protocol>://<hostname>/" from OUTPUT_URL
+            # and by prefixing "/" we will get absolute path to destination directory.
+            local dest_dir="/$(echo $OUTPUT_URL | cut -d/ -f4-)"
```

Comment:
I think the `local` variable setting should happen only once
outside of the `for` loop - just before the `for` loop

Reply:
Hmm, which `for` loop do you have on your mind? Because I don't see any ...

V.

Reply:
Ah, I just found it!
I've totally forgot the PXE stuff ...

Will fix that ;-)

V.

Reply:
Done!